weakmap.c

```diff
@@ -471,14 +565,27 @@ wmap_delete(VALUE self, VALUE key)
     }
 }
 
-/* Returns +true+ if +key+ is registered */
+/*
+ *  call-seq:
+ *    map.key?(key) -> true or false
+ *    map.include?(key) -> true or false
```

Comment:
Probably best to only include the `include?` line, because the other are aliases.

Reply:
My bad, I test-rendered the changes with older version of RDoc gem which wasn't yet be able to make the aliases docs just a link, so I used the old approach of providing every call-sequence (_now_ I remember we even discussed that before and the change was introduced, but :shrug: )